BeeSwift/Components/GoalImageView.swift

```diff
@@ -61,24 +61,37 @@ class GoalImageView : UIView {
             object: nil,
             queue: OperationQueue.main
         ) { [weak self] _ in
-            self?.refresh()
+            DispatchQueue.main.async {
+                self?.refresh()
+            }
```

Comment:
Could you explain why this is part of this PR?

Reply:
`.refresh()` has been marked `@MainActor` as part of this merge request which turned calling refresh here like this into a warning now and error in Swift 6.

> Call to main actor-isolated instance method 'refresh()' in a synchronous nonisolated context; this is an error in the Swift 6 language mode

![Screenshot 2025-01-09 at 11 17 58](https://github.com/user-attachments/assets/61defc1a-3469-4f01-90e3-fb307671c768)

Reply:
Oh Swift. So `DispatchQueue.main` is ok, but `OperationQueue.main` is not.